src/language-markdown/printer-markdown.js

```diff
@@ -842,6 +754,20 @@ function printFootnoteReference(node) {
   return `[^${node.label}]`;
 }
 
+/**
+ * @param {AstPath} path
+ * @param {*} options
+ * @param {*} print
+ * @returns {Doc}
+ */
+function printParagraph(path, options, print) {
+  const parts = [];
+  path.each(() => {
+    parts.push(print());
+  }, "children");
```

Comment:
```suggestion
  const parts = path.map(print, "children")
```

Maybe we should add an ESLint rule for this.

Reply:
Oh I wonder why I avoided `path.map()` though I love declarative style. I have thought I saw a comment like "map is deprecated" but it is just a phantom... Thank you to point it out.